examples/nlp/intent_slot_classification/conf/intent_slot_classification_config.yaml

```diff
@@ -0,0 +1,96 @@
+# Sentence classification with pretrained BERT models
+
+trainer:
+  gpus: 1 # the number of gpus, 0 for CPU
+  num_nodes: 1
+  max_epochs: 100
+  max_steps: null # precedence over max_epochs
+  accumulate_grad_batches: 1 # accumulates grads every k batches
+  amp_level: O0 # O1/O2 for mixed precision
+  precision: 16 # Should be set to 16 for O1 and O2, default is 16 as PT ignores it when am_level is O0
+  distributed_backend: ddp
+  row_log_interval: 1  # Interval of logging.
+  val_check_interval: 1.0  # Set to 0.25 to check 4 times per epoch, or an int for number of iterations
+  resume_from_checkpoint: null # The path to a checkpoint file to continue the training, restores the whole state including the epoch, step, LR schedulers, apex, etc.
+
+  checkpoint_callback: False  # Provided by exp_manager
+  logger: False  # Provided by exp_manager
+
+model:
+  data_dir: ??? # /path/to/data
+  class_balancing: null # or weighted_loss
+  intent_loss_weight: 0.6 # relation of intent to slot loss in total loss (between 0 to 1)
+  pad_label: '-1'
+  none_slot_label: 'O'
+  ignore_extra_tokens: false
+  ignore_start_end: false
+
```

Comment:
please check the defaults here, in your dataset the default for pad_label is 128, do you actually use these config values in the model?

Reply:
Ok, corrected it